### PR TITLE
[MaterializedView] Generate MaterializedView Full DDL SQL for ShowCreateTableStmt and ShowMaterializedViewStatement 

### LINKED ISSUE #8396: [MaterializedView] Generate MaterializedView Full DDL SQL for ShowCreateTableStmt and ShowMaterializedViewStatement
## Feature request

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

show full DDL SQL for new MaterializedView , so that it can be used to create mv directly

### PR DESCRIPTION
**Pull request by Crystal-LiuJing**
## What type of PR is this：
- [ ] bug
- [x] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #8396

## Problem Summary(Required) ：

1 show full ddl_sql for new MaterializedView

### CODE REVIEW

**Comment by Crystal-LiuJing** [comment]
> There are a lot of UTs that may need to be fixed.



> There are a lot of UTs that may need t

**Comment by Crystal-LiuJing** [comment]
> There are a lot of UTs that may need to be fixed.

yes , i'll check it

**Comment by HangyuanLiu** [comment]
toSql is an obsolete interface and cannot be used in actual functional code. Relevant code can be moved into AST2SQL

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 57 / 65 (87.69%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/service/FrontendServiceImpl.java|0|4|00.00%|
|:large_blue_circle:|com/starrocks/catalog/ExpressionRangePartitionInfo.java|20|23|86.96%|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|37|38|97.37%|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 55 / 62 (88.71%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/service/FrontendServiceImpl.java|0|4|00.00%|
|:large_blue_circle:|com/starrocks/catalog/ExpressionRangePartitionInfo.java|20|23|86.96%|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|35|35|100.00%|

**Comment by Astralidea** [comment]
run starrocks_fe_unittest

**Comment by Astralidea** [comment]
run starrocks_fe_unittest

**Comment by Astralidea** [comment]
run starrocks_fe_unittest

**Comment by Astralidea** [comment]
run starrocks_fe_unittest

File: fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java [hunk 1]
```diff
@@ -1849,7 +1854,63 @@ public static void getDdlStmt(String dbName, Table table, List<String> createTab
...
+
+            // refresh schema
+            MaterializedView.MvRefreshScheme refreshScheme = mv.getRefreshScheme();
+            sb.append("\nREFRESH ").append(refreshScheme.getType());
+            if (refreshScheme.getType() == MaterializedView.RefreshType.ASYNC) {
+                MaterializedView.AsyncRefreshContext asyncRefreshContext = refreshScheme.getAsyncRefreshContext();
+                sb.append(" START(\"").append(Utils.getDatetimeFromLong(asyncRefreshContext.getStartTime())
+                                .format(DateUtils.DATE_TIME_FORMATTER))
+                        .append("\")");
+                if (asyncRefreshContext.getTimeUnit() != null) {
```
Comment (ABingHuang) [inline fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java:1880]: should consider the situation of 
REFRESH ASYNC without interval.

**Review by ABingHuang — CHANGES_REQUESTED** [review]

**Comment by Crystal-LiuJing** [comment]
run starrocks_fe_unittest

**Review by Astralidea — APPROVED** [review]

**Review by ABingHuang — APPROVED** [review]
LGTM

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Review by chaoyli — APPROVED** [review]

**Pull request merged**